packages/frontend/src/hooks/echarts/useEchartsFunnelConfig.ts

```diff
@@ -54,22 +73,39 @@ const useEchartsFunnelConfig = (isInDashboard: boolean) => {
             tooltip: {
                 trigger: 'item',
                 formatter: ({ marker, name, value }) => {
-                    const formattedValue = formatItemValue(
-                        selectedField,
-                        value,
-                    );
-
-                    const percentOfMax = round(
-                        (Number(value) / chartConfig.maxValue) * 100,
-                        2,
-                    );
-
-                    return `${marker}<b>${name}</b><br /> Value: ${formattedValue} <br/> Percent of start: ${percentOfMax}%`;
+                    const { formattedValue, percentOfMax } =
+                        getValueAndPercentage({
+                            field: selectedField,
+                            value,
+                            maxValue: chartConfig.maxValue,
+                        });
+
+                    return `${marker}<b>${name}</b><br /> Value: ${formattedValue} <br/> Percent: ${percentOfMax}%`;
                 },
             },
             label: {
                 show: true,
-                position: 'inside',
+                position: label?.position || FunnelChartLabelPosition.INSIDE,
```

Comment:
we could add here `color: 'black'` if the position is different from `inside`?

Reply:
Done